Frameworks/Foundation/NSURLProtocol_WinHTTP.mm

```diff
@@ -251,7 +251,7 @@ - (void)startLoading {
 
         NSError* error = nil;
         try {
-            auto httpRequestMessage = _requestMessageForNS(_request);
+            auto httpRequestMessage = _requestMessageForNS(self.request);
```

Comment:
(the compiler can't perform common subexpression elimination as it can't prove that `self.request` is idempotent)